### ISSUE TITLE
I cannot get the express-ts example to work as is

### ISSUE THREAD

**Issue by mKurrels** [Vincit/objection.js#652]
When I run npm start I get:

> objection-example-express-ts@0.7.0 start /Users/mike/workspace/node/objection.js/examples/express-ts
> tsc && node lib/app.js

../../typings/objection/index.d.ts(6,23): error TS2307: Cannot find module 'knex'.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! objection-example-express-ts@0.7.0 start: `tsc && node lib/app.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the objection-example-express-ts@0.7.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mike/.npm/_logs/2017-12-09T23_01_56_645Z-debug.log

**Comment by elhigu** [comment]
Sounds like you are missing package `@types/knex`.

**Comment by koskimas** [comment]
@elhigu Nope, there is something wrong with the example. @mKurrels Thanks for bringing this up. @mceachen Could you take a look what's wrong with the example project when you have time?

**Comment by mKurrels** [comment]
So as far as I can tell it has something to do with the examples/express-ts folder being inside of the objection folder. There are two different node modules, one in the top level folder and one in the express-ts folder. And the types are in the top level folder. There's some sort of conflict between the two node_modules folders and the typings folder. Not sure what the issue is.

But I was able to find a work around where I moved the example folder outside of the objection folder, removed objection from package.json (as it was using a relative path), ran npm install --save objection, and copied the typings folder into the example folder.

Then the application works for me. (hope this helps someone that knows more about this stuff than me).
